package.json

```diff
@@ -20,7 +20,8 @@
     "docgen": "lerna run docgen",
     "docker-build:app": "yarn workspace example-app build && docker build . -t spotify/backstage",
     "docker-build": "yarn tsc && yarn workspace example-backend build-image",
-    "create-plugin": "backstage-cli create-plugin",
+    "create-plugin": "backstage-cli create-plugin --scope internal",
```

Comment:
In this repo we'd just have a single `create-plugin`, with `--scope backstage --no-private`

Reply:
ok